atomicapp/cli/main.py

```diff
@@ -49,6 +49,7 @@ def set_arguments(self):
         parser_install = subparsers.add_parser("install")
 
         parser_install.add_argument("--no-deps", dest="nodeps", default=False, action="store_true", help="Skip pulling dependencies of the app")
+        parser_install.add_argument("--answers-format", dest="answers_format", default=ANSWERS_FILE_SAMPLE_FORMAT, help="The format for the answers.conf.sample file.Default is 'ini', Valid formats are 'ini', 'json', 'xml', 'yaml'.")
```

Comment:
Please move this option to global parser (i.e. `atomicapp --answers-format ...` not `atomicapp install --answers-format` as `answers.conf.sample` is generated also during `run` if there is no answers file found.